.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ default_job: &default_job
         command: cd build && ctest --output-on-failure
     - run:
          name: Run benchmarks
-         command: cd build && make benchmarks
+         command: cd build && make install && make benchmarks
```

Comment:
why do you need make install? Shouldn't it be possible to run the benchmarks without installing?

Reply:
I added python benchmarks, and for these I need the python rascal package.

Reply:
Yes, but could you change the Python benchmarks to not require make install? Perhaps using the same PYTHONPATH trick as #165.

Reply:
Feel free to change this. I am not sure what the PYTHONPATH trick in this pull request is.

Reply:
https://github.com/cosmo-epfl/librascal/blob/23a5d261951f9bb6e954dd4f68359fb6c3d26be7/tests/CMakeLists.txt#L32-L35